database/migrations/2023_02_08_102837_create_ec_pois_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('ec_pois', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->string('name');
+            $table->text('description')->nullable();
+            $table->lineString('geometry')->nullable();
```

Comment:
POI significa Point Of Interest: si rappresenta con un punto e con una linestring.

Reply:
con un punto e NON con una linestring